pkg/controllers/federatedhpa/federatedhpa_controller.go

```diff
@@ -101,6 +101,27 @@ type timestampedRecommendation struct {
 	timestamp      time.Time
 }
 
+// hpaReqParams is parameters to get recommend metrics
+type hpaReqParams struct {
+	hpa            *autoscalingv1alpha1.FederatedHPA
+	metricSpec     autoscalingv2.MetricSpec
+	selector       labels.Selector
+	metricSelector labels.Selector
+	podList        []*corev1.Pod
+	calibration    float64
```

Comment:
It would be helpful to provide some clarification on the meaning of calibration and how it is calculated.